src/core/component.test.ts

```diff
@@ -912,6 +912,30 @@ describe("template", () => {
         expect(component.$children.length).toBe(1);
     });
 
+    it("updates templates on component updating (check via functional component inside)", () => {
+        expect.assertions(2);
+        const FunctionalComponent = Vue.extend({
+            functional: true,
+            render(h) {
```

Comment:
Is it possible to use the Jest `spyOn` function to assert the `render` method called twice?

Reply:
Or even better - assert it was called after the `$forceUpdate` call

Reply:
I'll try